compile error under g++-8.2

g++ 8.2 spits out the following error:
```
VpiImpl.cpp: In member function ‘virtual GpiIterator* VpiImpl::iterate_handle(GpiObjHdl*, gpi_iterator_sel_t)’:
VpiImpl.cpp:474:12: error: ‘new_iter’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     return new_iter;
            ^~~~~~~~
```

it can be fixed with the following change:

```patch
diff -ru cocotb-1.1/lib/vpi/VpiImpl.cpp coco/lib/vpi/VpiImpl.cpp
--- cocotb-1.1/lib/vpi/VpiImpl.cpp	2019-01-24 05:31:07.000000000 +0800
+++ coco/lib/vpi/VpiImpl.cpp	2019-02-04 00:42:21.254449248 +0800
@@ -456,7 +456,7 @@
 
 GpiIterator *VpiImpl::iterate_handle(GpiObjHdl *obj_hdl, gpi_iterator_sel_t type)
 {
-    GpiIterator *new_iter;
+    GpiIterator *new_iter = NULL;
     switch (type) {
         case GPI_OBJECTS:
             new_iter = new VpiIterator(this, obj_hdl);
```

--- a/lib/vpi/VpiImpl.cpp
+++ b/lib/vpi/VpiImpl.cpp
@@ -456,7 +456,7 @@ GpiObjHdl *VpiImpl::get_root_handle(const char* name)
 
 GpiIterator *VpiImpl::iterate_handle(GpiObjHdl *obj_hdl, gpi_iterator_sel_t type)
 {
-    GpiIterator *new_iter;
+    GpiIterator *new_iter = NULL;
     switch (type) {
         case GPI_OBJECTS:
             new_iter = new VpiIterator(this, obj_hdl);

